iroh/src/commands.rs

```diff
@@ -567,6 +567,27 @@ pub enum BlobCommands {
     /// Delete content on the node.
     #[clap(subcommand)]
     Delete(self::delete::Commands),
+    /// Get a ticket to share this blob.
+    Share {
+        /// Hash of the blob to share.
+        hash: Hash,
+        /// Add this token to the ticket.
```

Comment:
include an optional authentication token in the ticket

Reply:
done in [f4cbea3](https://github.com/n0-computer/iroh/pull/1746/commits/f4cbea3492b16b9de1bb4a9fe95ef7b181aedbf6)